src/_shared/components/TabLink/TabLink.test.tsx

```diff
@@ -35,7 +35,7 @@ describe('The TabLink component', () => {
   it('renders with article count that is larger than 1000', () => {
     render(
       <MemoryRouter>
-        <TabLink count={1235} tabSelected to="/any/path/">
+        <TabLink count={1235} isCurrentTab to="/any/path/">
```

Comment:
This was giving me an error that `tabSelected` is not a valid prop, turns out they changed the prop name?

Reply:
Nice! 👍